File: Run8/LibRun8/Formats/Terrain/Tr3.cs
namespace LibRun8.Formats.Terrain
{
    public class Tr3 : TerrainTile
    {
        public Tr3(string path, string region, float heightOffset, BinaryReader reader)
        {
            this.region = region;
            this.reader = reader;
            this.heightOffset = heightOffset;
            throw new NotImplementedException();
        }

        public void Write(string path)
        {
            throw new NotImplementedException();
        }
    }
}

File: blender_addon/terrain/tr3.py
from ..binreader import BinaryReader
from .terrain_utils import ETileType, TerrainTile


class TR3(TerrainTile):
    def __init__(self, reader: BinaryReader, x: float, y: float) -> None:
        super().__init__(reader, ETileType.TR3, x, y)

        self.read()

    def read(self):
        pass
